pages/authentication/signin.html

```diff
@@ -32,7 +32,7 @@ <h2 class="nav-logo">WitTest</h2>
             <input class="input nav-search" type="text" placeholder="Search" />
```

Comment:
as an alternative, we can also give type="search" for search input. 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/search

Reply:
Thanks for the suggestion, I will check it out.